Foosball/Foosball/TournamentBracket.cs

```diff
@@ -13,88 +13,27 @@ namespace Foosball
 {
     public partial class TournamentBracket : Form
     {
-        private int _amount;
-        private List<string> _teamNames = new List<string>();
-        private int _round;
-
-        public TournamentBracket(List<String> teamNames, int round = 1)
+        private Tournament _tournament;
+        public TournamentBracket(Tournament tournament, int round = 1)
         {
             if(round == 1) {
-                var shuffledTeamNames = teamNames.OrderBy(a => Guid.NewGuid()).ToList();
-                teamNames = shuffledTeamNames;
+                var shuffledTeamNames = tournament.Players.OrderBy(a => Guid.NewGuid()).ToList();
+                tournament.Players = shuffledTeamNames;
             }
-
-            _teamNames = teamNames;
-            _round = round;
-
-            CheckIfEnded();
+            _tournament = tournament;
+            TournamentBracketMethods methods = new TournamentBracketMethods(tournament, this);
```

Comment:
I think this class should be named TournamentController and be stored in Controllers folder